pkg/ebpf/bytecode/perf.go

```diff
@@ -0,0 +1,27 @@
+//+build linux_bpf
+
+package bytecode
+
+import (
+	"github.com/DataDog/ebpf/manager"
+)
+
+type ClosedConnPerfHandler struct {
```

Comment:
Nothing about this seems specific to closed conns re: types defined below. Should it just be called a `PerfHandler`?